src/selections/selection-position.js

```diff
@@ -56,7 +56,7 @@ const centerToolbar = function(toolbar, rect) {
 		if (startContainer.$.nodeType !== Node.ELEMENT_NODE) {
 			startContainer = startContainer.getParent();
 		}
-		if (startContainer) {
+		if (startContainer.$.nodeName === "TD" || startContainer.$.nodeName === "TH") {
```

Comment:
It looks like this actually fixes a small bug in `master`, where a text line (paragraph) followed by an image causes wrong positioning. Awesome! I'll trust you that this also fixes the 7.3 bug from the ticket.

Before merging, we should just tidy this a bit. Please move node name to a variable, use single quotes, break line if needed. If you followed [setup](https://github.com/liferay/alloy-editor/wiki/Contributing#setup), you can use `yarn format` (and `yarn test` for doublecheck).